_gcode/M207.md

```diff
@@ -47,3 +47,14 @@ parameters:
 ---
 
 Set lengths, feedrate, and Z lift for firmware-based retraction. See parameters below.
+
+### Retract/Recover tuning
+
+Typically one of these cases where the correct combination of 4 parameters is required for a good tuning. This is obviously tricky but JCZD wrote a test pattern to help with this while using a little filament as possible:
+
+![retract-recover helper](https://engrnage.ch/files/marlin_retract_recover_helper.png)
+
+* [3x3x3x3 pattern](https://engrnage.ch/files/retract_recover_3x3x3x3-jczd.gcode) - 81 individual samples, you probably want to start with this and it should give you a decent approximation
+* [4x4x4x4 pattern](https://engrnage.ch/files/retract_recover_3x3x3x3-jczd.gcode) - 256 individual samples, for fine-tuning
```

Comment:
Files and images need to be copied locally into the site. At the moment these links are broken.